lib/collection/src/operations/universal_query/shard_query.rs

```diff
@@ -397,6 +397,7 @@ impl TryFrom<grpc::Expression> for ExpressionInternal {
                     .map_err(|_| tonic::Status::invalid_argument("invalid payload key"))?;
                 ExpressionInternal::GeoDistance { origin, to }
             }
+            Variant::Datetime(dt_str) => ExpressionInternal::DateTime(dt_str),
```

Comment:
It looks like we have two different formats: `DateTime` and `Datetime`

I prefer the first everywhere if possible, as it's much more commonly used.

Reply:
At the same time, we also have `DatetimeRange` and `FieldTypeDatetime` in grpc, and their counterparts in REST. This is why I stuck with the single word representation. 

I can homogenize into `Datetime` in the internal variants too.